release/rancher/rancher.go

```
@@ -311,6 +322,158 @@ func formatContentLine(line string) string {
 	return strings.TrimSpace(line)
 }
 
+func GenerateMissingImagesList(version string, concurrencyLimit int) ([]string, error) {
+	if !semver.IsValid(version) {
+		return nil, errors.New("version is not a valid semver: " + version)
+	}
+	const rancherWindowsImagesFile = "rancher-windows-images.txt"
+	const rancherImagesFile = "rancher-images.txt"
+	rancherWindowsImages, err := getRancherPrimeArtifact(version, rancherWindowsImagesFile)
+	if err != nil {
+		return nil, errors.New("failed to get rancher windows images: " + err.Error())
+	}
+	rancherImages, err := getRancherPrimeArtifact(version, rancherImagesFile)
+	if err != nil {
+		return nil, errors.New("failed to get rancher images: " + err.Error())
+	}
+	images := append(rancherWindowsImages, rancherImages...)
+
+	// create an error group with a limit to prevent accidentaly doing a DOS attack against our registry
+	ctx, cancel := context.WithCancel(context.Background())
+	errGroup, _ := errgroup.WithContext(ctx)
```

Are we ignoring the error?